README.md

```diff
@@ -125,3 +125,20 @@ const route = {
 | `component`    | The component to be rendered given the required feature flag is true.               | `vue component`      |
 | `requiredFlag` | If the given feature flag is false, the user will be redirected to the given route. | `string`             |
 | `to`           | The path or object which vue router will push.                                      | `string` or `object` |
+
+## Development
+
+After cloning the repo to your machine:
+
+```bash
+$ npm install
+$ npm run watch
+```
+
+### Local
+
+If you wish to test out your changes in another project locally, you can install with your local path to vue-ld. Your vue app will detect changes so long as vue-ld is being watched.
```

Comment:
nit: Might want to clarify `install with your local path to vue-ld` with a link or example.